packages/messages/src/utils/constants.js

```diff
@@ -0,0 +1,8 @@
+const supportedEnvs = {
+  PRODUCTION: 'production',
+  DEVELOP: 'develop',
```

Comment:
Can you change develop to development?
```suggestion
  DEVELOPMENT: 'development'',
```